docs/getting-started.md

```diff
@@ -54,11 +54,19 @@ argo version
 You can download the latest and previous Argo binaries from our [releases page](https://github.com/argoproj/argo/releases/).
 
 ## 2. Install the Controller and UI
+
 ```sh
 kubectl create namespace argo
-kubectl apply -n argo -f https://raw.githubusercontent.com/argoproj/argo/v2.4.3/manifests/install.yaml
+kubectl apply -n argo -f https://raw.githubusercontent.com/argoproj/argo/v2.5.2/manifests/install.yaml
 ```
+
+Namespaced installs as well as installs with MinIO and/or a database built in [are also available](https://github.com/argoproj/argo/tree/v2.5.2/manifests). 
```

Comment:
@simster7 would it make sense to just use the namespaced install yaml rather than `-n argo`? Or am I misunderstanding what that yaml does?

Reply:
Namespaced install will limit the `workflow-contorller` to a single namespace, only allowing workflows to be run there. In general, users want to be able to run workflows in arbitrary namespaces, so I don't think it would make sense

Reply:
Ah got it. Thanks!